.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
```

Comment:
Will we add the rest of our pre-commit later? Also is there a way to share pre-commit files?

Reply:
Yes.

> no flake8/isort; flake8 and precommit-native hooks should be standardized across all python repos, and there hasn't been any decision regarding that yet.

And if by sharing you mean automagically cross-repo across the network, not that I know of. The config file has to be manually updated. This goes back to my quote above (why I'm hesitant about defining more pre-commit stuff)